e2e/utils/waits-utils.ts

```diff
@@ -274,5 +274,15 @@ export async function waitWhileLoading(page: Page, timeOut: number = 90000): Pro
   await page.waitForFunction((css) => {
     const elements = document.querySelectorAll(css);
     return elements && elements.length === 0;
-  }, {polling: 'mutation', timeout: timeOut}, spinElementsSelector);
+  }, {polling: 'mutation', timeout}, spinElementsSelector);
+}
+
+export async function waitUntilEnabled(page: Page, cssSelector: string): Promise<boolean> {
+  const jsHandle = await page.waitForFunction(xpathSelector => {
+    const elemt = document.evaluate(xpathSelector, document, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue;
+    const style = window.getComputedStyle(elemt as Element);
+    const propValue = style.getPropertyValue('cursor');
+    return propValue === 'pointer';
+  }, {}, cssSelector);
+  return (await jsHandle.jsonValue()) as boolean;
```

Comment:
This was part of button `waitUntilEnabled`. Instead of removing code, I'm saving code in a new function.